src/main/java/org/isf/menu/gui/UserEdit.java

```diff
@@ -340,20 +340,24 @@ private JComboBox<UserGroup> getUserGroupComboBox() {
 		if (userGroupComboBox == null) {
 			userGroupComboBox = new JComboBox<>();
 			try {
-				List<UserGroup> group = userBrowsingManager.getUserGroup();
+				List<UserGroup> groups = userBrowsingManager.getUserGroup();
 				if (insert) {
-					if (group != null) {
-						for (UserGroup elem : group) {
-							userGroupComboBox.addItem(elem);
+					if (groups != null) {
+						for (UserGroup group : groups) {
+							if(!group.isDeleted()) {
+								userGroupComboBox.addItem(group);
+							}
 						}
 					}
 				} else {
 					UserGroup selectedUserGroup = null;
-					if (group != null) {
-						for (UserGroup elem : group) {
-							userGroupComboBox.addItem(elem);
-							if (user.getUserGroupName().equals(elem)) {
-								selectedUserGroup = elem;
+					if (groups != null) {
+						for (UserGroup group : groups) {
+							if(!group.isDeleted()) {
+								userGroupComboBox.addItem(group);
```

Comment:
Isn't this else block used if `insert` is false?   Then why add the group to the combobox?  What am I mssing?

Reply:
I'm not understanding what you mean, sorry. I've just wrapped the instruction block with the 'if' statement.

Reply:
nevermind.......